src/Lime.Client.TestConsole/Lime.Client.TestConsole.csproj

```diff
@@ -85,16 +85,16 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Lime.Messaging, Version=0.8.70.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Those file paths for Lime packages inside `<HintPath>` must stay with `net461` in order to build (and run) TestConsole. 

Mainly because we can't build Transport projects for net462 too, since they target netstandard2.1, which is not compatible with any version of .Net Framework...